src/Commands/CreateUserCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Commands;
+
+use App\Exceptions\User\CreateUserErrorException;
+use App\Repository\User\UserManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CreateUserCommand extends Command
+{
+    private const DEFAULT_PASSWORD = 'ToBeChanged';
```

Comment:
What? Nobody can change your class ever! It should at least be a config.

Personally I'm dead against using set passwords, could you generate it display it on the terminal or something like that?

Reply:
Sure I can generate one on the spot and print it at the screen, or better, I will look on Symfony mail to send it directly to the new user by email

Reply:
Mails are not secured, so if you go that way you're going to work on the change password next, and also add a mechanism to force the user to change his/her/its password on first login.
Don't do it all here, open issues if you go down that way